### DISCUSSION TITLE
Route to statistics is not working for short codes with slashes

### DISCUSSION THREAD

**Discussion by WrdVdb** [shlinkio/discussions#25] [Help wanted] [answered]
#### Shlink web client version

* Version: 3.10.1
* How do you use shlink-web-client: self-hosted

#### Summary

We created a shortlink that contains slashes /test/123
The link works correct, but we cannot access the stats in de app:

![image](https://github.com/shlinkio/shlink-web-client/assets/36230484/f582443e-5300-4d88-9eae-c4d11b43a9e4)

#### Current behavior

Statistics page is not accessible when the short link contains slashes

#### Expected behavior

Show the stats when the short link contains slashes

#### How to reproduce

Create a short link that contains

**Comment by acelaya** [comment]
Are you sure you are using shlink-web-client 3.10.1? Because support for URLs with multi-segment slugs was [added in 3.7.2](https://github.com/shlinkio/shlink-web-client/issues/694)

Since that version, shlink-web-client parses the slug replacing `/` with `__` so that client-side routing works:

![image](https://github.com/shlinkio/shlink-web-client/assets/2719332/ead7b49d-01a6-46d7-b5b7-da5a86427a87)

If you built that URL manually yourself, then it is expected to not work. You have to do the parsing above.

**Comment by WrdVdb** [comment, marked as answer]
Ah, I'm using 3.5.2, not 3.10.1.
The webclient shows: "version: latest" and I assumed that I was using the latest version, but it's only a link to the latest version.
I will update my webclient! Thank you!

**Comment by acelaya** [comment]
Yeah, that probably needs some thinking.

The "latest" tag is built every time a change is merged into develop, but if you have an outdated "latest", it is actually not the true latest, if that makes any sense 😅

I'm thinking it should somehow also reference the latest stable version it was built upon.

That said, I would recommend not relying on "latest", as it can have unstable changes.

**Comment by WrdVdb** [comment]
I was confusing the server and client version. We are using the server version 3.5.2. I can't see the version of the web client anywhere. How can I check which web client version we are using?

**Reply by acelaya** [reply]
You can't, if you are using `latest` ~or `stable`~. I need to fix that.
